### PR TITLE
ci: Test 32-bit builds

### PR DESCRIPTION
**Pull request by kamalmarhubi**

### CODE REVIEW

**Comment by kamalmarhubi** [comment]
There are a couple of TODOs but this should be fine to merge as is. Once merged, I'll rebase #82 and then we should be able to merge it.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
+  # Script to test 32-bit versions.
+  - curl -sSL https://raw.githubusercontent.com/carllerche/travis-rust-matrix/master/test | bash
 
 matrix:
+  # TODO: Figure out weridness with `ARCH` being set twice.
   include:
   - rust: nightly
     script:
       - cargo test --features="dev nightly"
+  - rust: nightly
```
Comment (siddontang) [inline .travis.yml]: I don't known why two `nightly` here, One with `dev`, the other is not?

Comment (kamalmarhubi) [inline .travis.yml]: To ensure that nightly passes with the `AtomicUsize`-based counters being added in #82, as well as with `AtomicU64` from `#[feature(integer_atomics)]`.

Comment (kamalmarhubi) [inline .travis.yml]: Oh I see what you mean. I meant to have one with `nightly` and one without, and both with `dev`. Fixing it.

**Comment by siddontang** [comment]
LGTM
PTAL @overvenus

**Pull request merged**